src/Domain/LeanCode.DomainModels.EF/DateConverter.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+
+namespace LeanCode.DomainModels.EF
+{
+    public class DateConverter : ValueConverter<Date, DateTime>
+    {
+        public static readonly DateConverter Instance = new DateConverter { };
```

Comment:
`new DateConverter();`

Reply:
I thought we skipped `()` for parametersless constructors?

Reply:
IFF you set some properties using object initializer. If you have to choose between `new T {};` vs `new T();`, select the latter.